crates/bevy_render/src/color/mod.rs

```diff
@@ -487,6 +487,15 @@ impl Color {
             } => [hue, saturation, lightness, alpha],
         }
     }
+
+    pub fn as_linear_abgr_u32(self) -> u32 {
```

Comment:
Is this going to cause a loss of fidelity in our colors due to discretization?

Reply:
No change from before. We have already been doing this. It was just done in `prepare_sprites`. The color passed to the shader was already encoded like this.

I just moved that computation earlier to the Extract stage, so that the color can be stored the same way in the `ExtractedSprite` struct, too, making it smaller.